examples/redux-saga-typescript-graphql-fluentui/components/layouts/header.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import { getTheme, Text } from 'office-ui-fabric-react'
+
+const theme = getTheme()
+
+const renderDetailsRowStyles = {
+  root: {
+    backgroundColor: theme.palette.themePrimary,
+    color: theme.palette.white,
+    lineHeight: '50px',
+    padding: '0 20px',
+  },
+}
+
+export const Header: React.FunctionComponent = () => (
+  <Text as="h2" nowrap block styles={renderDetailsRowStyles}>
+    this is a heder
```

Comment:
The image in the readme also has `heder` and needs the same fix.

```suggestion
    this is a header
```